### ISSUE TITLE
Group Attribute Order not being honored in v7

### ISSUE THREAD

**Issue by winmastr** [SparkDevNetwork/Rock#2678]
Description: When adding new Group Attributes to a GroupTypeDetail, the order in which they are arranged in the groupTypedetail tool is being ignored when presented within the actual Group. I verified in SQL that the ORDER number is correct but the UI is ignoring the changes.

* **Rock Version:** [7

**Comment by edmistj** [comment]
Can someone replicate this issue as a confirmation. Someone on our team tried and was not able to reproduce it.

**Comment by winmastr** [comment]
The issue appears to be a non-issue as I initially described it. The ignoring ordering on the GroupTypeDetail page under Group Attributes appears to follow 1st Category then unassigned attributes to a category. In my example, the 6 new attributes were being scattered throughout the listed attributes on the GroupViewer Page for that Type of Group. I had looked at SQL to see what the order was reflecting and it was showing the order I had placed them in. It was discovered later that the newly added attributes were not part of the same attribute category the previous ones were apart of. When we included the 6 new attributes in that category, the sort order was corrected.
